zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -4475,6 +4475,37 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
       x-eventually-consistent: true
+
+  /message-subscriptions/statistics/process-definitions:
+    post:
+      tags:
+        - Message subscription statistics
+      operationId: getMessageSubscriptionProcessDefinitionStatistics
+      summary: Get message subscription statistics for process definitions
+      description: |
+        Get message subscription statistics for process definitions based on given criteria.
+      requestBody:
+        required: false
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/MessageSubscriptionProcessDefinitionStatisticsQuery"
```

🔧 We should switch the entities in the component names.
```suggestion
              $ref: "#/components/schemas/ProcessDefinitionMessageSubscriptionStatisticsQuery"
```